pkg/metainfo/kvmetainfo/objects.go

```diff
@@ -272,7 +272,7 @@ func (db *DB) getInfo(ctx context.Context, bucket string, path storj.Path) (obj
 
 	lastSegmentMeta := segments.Meta{
 		Modified:   pointer.CreationDate,
-		Expiration: convertTime(pointer.GetExpirationDate()),
```

Comment:
Is the convertTime still used in the package?

Reply:
No I deleted it